tutorials/visualization-with-gaia/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+matplotlib
+astropy
+astroquery
```

Comment:
```suggestion
astroquery>=0.4.8.dev9474  # 2024-09-24 pinned for Gaia column capitalization issue
```
This is just to be consistent with other tutorials using astroquery that encountered issues when executing the notebook.